app/lib/actions/katello/content_view_environment/reassign_objects.rb

```diff
@@ -4,8 +4,8 @@ module ContentViewEnvironment
       class ReassignObjects < Actions::Base
         def plan(content_view_environment, options)
           concurrence do
-            content_view_environment.systems.each do |system|
-              plan_action(System::Reassign, system, options[:system_content_view_id], options[:system_environment_id])
+            content_view_environment.hosts.each do |host|
+              plan_action(Host::Reassign, host, options[:system_content_view_id], options[:system_environment_id])
```

Comment:
Is there even a host reassign task?  I see one for systems in but not for hosts.